scripts/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@conversationai/conversationai-crowdsource-scripts",
+  "description": "Common scripts for working with data for crowdsourcing.",
+  "repository": "https://github.com/conversationai/crowdsource",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "tsc --outdir build/js-bin/",
+    "build:watch": "tsc --outdir build/js-bin/ --watch",
+    "test": "mocha build/js-bin/*_test.js"
```

Comment:
Just curious why mocha over karma now?

Reply:
Karma is a 'test runner' you can use with different testing frameworks (e.g. Jasmine, Mocha, etc), Mocha also supports standalone testing, so easier to use for Mocha standalone for testing like this I think. 

Now you ask, it does make me wonder if here we shouldn't be using [`ts-mocha`](https://www.npmjs.com/package/ts-mocha), and skip the compilation step (or rather do it inline)... I think I'll do that in a followup.

Reply:
Interesting, thanks for explaining!